### PR TITLE
Type declarations

### PR DESCRIPTION
**Pull request by aminya**
Updated types for tool-bar

PS: added a 14px option so later I make a PR for that one: https://github.com/suda/tool-bar/issues/288

### CODE REVIEW

**Comment by suda** [comment]
Nice! It's probably best to merge this in after all the other PRs get in :+1:

**Comment by suda** [comment]
@aminya probably needs to be rebased on top of `master` now :)

**Comment by aminya** [comment]
This is ready

**Comment by aminya** [comment]
@suda I ran the CI again. This is ready.

File: @types/index.d.ts [hunk 1]
```diff
@@ -1,9 +1,10 @@
 // tool-bar 1.2.x
 // https://atom.io/packages/tool-bar
+// Definitions by: aminya <https://github.com/aminya>
 
 /// <reference path="./config.d.ts" />
 
-import { TooltipManager } from "../index"
+import {TooltipPlacement, Disposable} from "atom" // "../index";
```
Comment (ericcornelissen) [inline @types/index.d.ts:7]: Small nitpick, but I'd suggest removing this comment

```suggestion
import {TooltipPlacement, Disposable} from "atom";
```

Comment (aminya) [reply @types/index.d.ts:7]: This is a reminder for creating PRs to the actual `@types/atom`. It should be `"../index"` for that repository. I prefer to keep it.

**Comment by suda** [comment]
It's conflicting after merging in the refactor :(

**Comment by aminya** [comment]
> It's conflicting after merging in the refactor :(

I fixed the conflict.

**Pull request merged**